[NBL-970]: Add approvals workflow icons and fix switch component styling

teogrosu opened the pull request:
Adds 4 new icons and updates the background color for the switch component

https://www.figma.com/design/PmBXl6UuDCtjzTDOGMybwl/%F0%9F%91%80-n-eyes?node-id=5664-174571&t=rD17q9yBPhJVRR3u-4

https://www.figma.com/design/PmBXl6UuDCtjzTDOGMybwl/%F0%9F%91%80-n-eyes?node-id=5666-173954&t=rD17q9yBPhJVRR3u-4

You can see the icons being used here https://github.com/Ebury/eburyonline/pull/2876

teogrosu committed:
[NBL-970]: Add approvals workflow icons and fix switch component styling

teogrosu committed:
[NBL-970]: Bump version and change switch color

teogrosu committed:
[NBL-970]: Add icons snapshots

AfrazHussain reviewed (commented):
@teogrosu  Thanks. Looks good to me, but can you remove the cypress downloads.html file if that's not required?

teogrosu commented:
> @teogrosu Thanks. Looks good to me, but can you remove the cypress downloads.html file if that's not required?

yup, removed, thanks - I think I added it by mistake

AfrazHussain reviewed (approved):
Thanks!
